LMMS/lmms
# adding sample exporter

closes #7452

This Pull request adds sample exporting to lmms that is independent from AudioEngine.

How to test:
Open lmms, load a sample into a sample clip. Open the context menu and click on the new option called "export sample buffer". A file dialog should open up. Select an audio file to export to, or create a new file by using a custom name in the file dialog. Click "open" inside the file dialog. The sample should be exported to the selected output file.

related pull request: #7557
this allows #5990 to be finished
